package.json

```diff
@@ -13,16 +13,19 @@
     "build": "tsc",
     "gen-docs": "typedoc --readme none --mode modules --excludeNotExported --excludePrivate --exclude **/*_test.ts --out ./docs/api .",
     "pretest": "npm run posttest; ln -s node_modules bower_components",
-    "test": "wct -l chrome",
+    "test": "npm run lint && wct -l chrome",
     "posttest": "rm -f bower_components",
-    "checksize": "uglifyjs lit-html.js -mc --toplevel | gzip -9 | wc -c"
+    "checksize": "uglifyjs lit-html.js -mc --toplevel | gzip -9 | wc -c",
+    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i",
```

Comment:
I think this PR is missing the .clang-format file

Reply:
Ahhh, much better!